lib/classes/PluginManager.js

```diff
@@ -114,21 +119,27 @@ class PluginManager {
       return;
     }
 
-    this.loadCommands(pluginInstance);
+    this.loadCommands(pluginInstance, options);
     this.loadHooks(pluginInstance);
     this.loadVariableResolvers(pluginInstance);
 
     this.plugins.push(pluginInstance);
   }
 
   loadAllPlugins(servicePlugins) {
+    const internalPluginOptions = { isExternal: false };
+    const externalPluginOptions = { isExternal: true };
```

Comment:
Let's not create this variables, we can simply hardcode `{ isExternal: true }` as second argument when adding external plugins and in other cases do not pass any _options_ at all (as _options_ are _optional_)